content/docs/localization/joining-the-localization-team.md

```diff
@@ -7,40 +7,109 @@ aliases:
 ---
 
 {{< admonition info >}}
+If you found a **translation mistake/error** in Scratch Addons, please [send feedback](/feedback) about it.
+{{< /admonition >}}
+
+Scratch Addons is available in more than 15 languages thanks to the help of volunteers around the world.
+
+## Requirements to join as a translator
 
-_Already translated: **Italian, Japanese, Portuguese, Slovenian and Spanish**. We won't be accepting any new translators for these languages._
+* You must be **FLUENT IN THE ENGLISH LANGUAGE**.
+* You must be **a NATIVE SPEAKER** of the language you want to translate to.
+* You must have an email address which you check at least once a week.

```

Comment:
Hmm, what's this for? To get Transifex announcements? (On second thought, we should also utlize Discord better for this kind of announcements and comms regarding translation)

Reply:
> To get Transifex announcements

Yeah, pretty much. It also sets some sort of expectation at the top of the document regarding ideal translator activity.

Reply:
> we should also utlize Discord better for this kind of announcements and comms regarding translation

Most translators don't join the dev server, we should probably try to push that too.